source/adminguide/virtual_machines.rst

```diff
@@ -827,6 +827,8 @@ To delete a virtual machine:
 
 #. Click the Destroy Instance button. |Destroyinstance.png|
 
+#. Optionally, select whether to expunge the VM, or to delete any attached volumes.
```

Comment:
This reads a little like the user must *either* expunge the VM *or* delete the volumes, and that the user must do one of them and can't do both of them.

Reply:
there is a comma there, is that not good enough? Law suits have been won over a comma, so maybe not. it doesn't read as an _either_ to me, though.

Reply:
The aim of documentation is not to be good enough for someone to 'get the idea' but to be accurate, clear and concise. I pointed out it _could_ be misinterpreted and therefore a slight change would benefit users.

Reply:
I would say that depending on the day I could read as"either/or".

Is this documentation regarding the UI? I do recall an "expunge VM" option when destroying a VM, but  I do not have any recollection of "destroy attached volumes" check box. Is it something new?

Reply:
works for me.

Reply:
@rafaelweingartner yes
@houthuis c&p cut opout for you ;)